### DISCUSSION TITLE
DISPONIBILIDAD PARA UNIRME A UN EQUIPO

### DISCUSSION THREAD

**Discussion by alvezabimael** [Launch-X-Latam/MisionFrontEnd#508] [Team Posts]
Disponible por acá para formar parte de algún equipo para la hackaton!! dejo mi correo: [lx00128@innovaccion.mx], mi nombre es ABIMAEL ALVEZ si gustan encontrarme en Teams

**Comment by CAMILOITT** [comment]
mi correo para hacer equipo : lx11002@innovaccion.mx
